Bindings~/python/alttester/altobject.py

```diff
@@ -145,6 +145,29 @@ def get_all_components(self):
 
         return commands.GetAllComponents.run(self._connection, self)
 
+    def wait_for_component_property(self, component_name, property_name,
+                                    property_value, assembly,  timeout=20, interval=0.5):
+        """Returns the value of the given component property.
+
+        Args:
+            component_name (:obj:`str`): The name of the component. If the component has a namespace the format should
+                look like this: ``"namespace.componentName"``.
+            property_name (:obj:`str`): The name of the property of which value you want. If the property is an array
+                you can specify which element of the array to return by doing ``property[index]``, or if you want a
+                property inside of another property you can get by doing ``property.subProperty``.
+            property_value(:obj:`str`): The value of the component expected
+            assembly (:obj:`str`): The name of the assembly containing the component.
+            timeout (:obj:`int`, optional): The timeout for wait duration.
+            interval (:obj:`float`, optional): The interval.
+        Returns:
```

Comment:
You should leave an empty line between the `Args` and `Returns` blocks. 

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html